Add guidance for additional issue resolution details (#454)
medic/cht-docs: content/en/contribute/code/workflow.md

A good workflow would be to work locally, pushing to a remote branch as you make progress, possibly open a draft PR for some initial collaboration on tricky parts, and once everything is done, convert the draft PR to a regular PR to be reviewed.

> ### Updating The Issue With What You Actually Did

Reviewer: Do you think before code review is the right place to add the comment? I'd be tempted to do it before AT because the notes/screenshots/etc may change during review, and presumably the reviewer is technical enough not to need it.

Reviewer: This would also be a good time to remind the dev to add the "Breaking" and "UI/UX" labels as appropriate.

> Put something on the GitHub issue with what the final change actually was. This is important for multiple cases including:

Reviewer: ```suggestion
Add a comment to the GitHub issue with what the final change actually was. This is important for multiple cases including:
```

- Non-technical people may not understand the conversation thread on the issue. GitHub is a place that developers work, but it is also common to send non-technical people links to issues in GitHub.
- The QA team should have a quick way to know where to start testing.
- Issues with a lot of discussion of alternative solutions need a clear resolution and indication of which route was taken.

Options for doing this:
- Attach a short video - these are usually very well received and can often help people understand what happened much more clearly than a text description.
- Screenshots - pictures with big arrows on them can quickly convey important things to look at. If you start to need multiple screenshots consider the video option instead.

> - Write up a few sentences - Be sure to consider a non-technical audience when writing this.

Reviewer: ```suggestion
- Write up a few sentences - be sure to consider a non-technical audience when writing this.
```

> 

Reviewer: May be worth including an example/template, something like.

```
### Acceptance testing

1. Install branch: `my-branch`
2. y

### What was actually built

[video|screenshots|text]

### Documentation

- [link](url)
```
